agent/config/config.go

```diff
@@ -623,6 +647,7 @@ func (cfg *Config) String() string {
 			"PollingMetricsWaitDuration: %v, "+
 			"ReservedMem: %v, "+
 			"TaskCleanupWaitDuration: %v, "+
+			"TaskResourceWaitTimeout: %v"+
```

Comment:
Q: Should there be a `,` here before `%v` and `"+`?

Reply:
good catch, updated!